Dockerfile

```diff
@@ -38,5 +37,5 @@ USER www-data
 HEALTHCHECK --interval=10s --timeout=3s \
     CMD curl -f http://localhost:$PULSE_SERVER_PORT/healthcheck || exit 1
 
-ENTRYPOINT ["brownie"]
-CMD ["run", "depositor"]
+ENTRYPOINT ["python3"]
+CMD ["src/depositor.py"]
```

Comment:
@lidofinance/lido-tooling 
What do you think about the idea of taking main.py or main.ts as an entry point in projects?